{"name": "test_sample2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 6 == 4", "trace": "def test_sample2():\n>       assert 6==4\nE       assert 6 == 4\n\ntest_lab003.py:6: AssertionError"}, "start": 1702899384971, "stop": 1702899384971, "uuid": "34db5f94-a11a-405e-baa0-021db4a0e3ca", "historyId": "90a24967383e0dd8bf96bc3cda9aad39", "testCaseId": "90a24967383e0dd8bf96bc3cda9aad39", "fullName": "test_lab003#test_sample2", "labels": [{"name": "suite", "value": "test_lab003"}, {"name": "host", "value": "LAPTOP-BQVK3RU3"}, {"name": "thread", "value": "11096-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_lab003"}]}